ereolen_support/ereolen_material.form.inc

```diff
@@ -86,20 +86,23 @@ function ereolen_material_form($form, &$form_state) {
  * @throws \Exception
  */
 function ereolen_material_form_submit($form, &$form_state) {
-  global $user;
   $submission_text = variable_get('ereolen_material_submission_text');
 
-  // Check for and create user in Jira
-  $jira_user = _ereolen_support_search_jira_user($user, $form_state, $form_state['values']['ereolen_material_email']);
-  if (!$jira_user) {
-    $jira_user = _ereolen_support_create_jira_user($user, $form_state, $form_state['values']['ereolen_material_email']);
+  // Search for a user.
+  $jiraUserId = _ereolen_search_user($form_state['values']['ereolen_material_email']);
+
+  // Create user if none are found.
+  if (empty($jiraUserId)) {
+    $jiraUser = _ereolen_support_create_customer($form_state['values']['ereolen_material_email']);
+    $jiraUserId = $jiraUser['content']->accountId;
```

Comment:
We'll get a runtime error here if calling `_ereolen_support_create_customer` fails (or if the result for some reason does not contain a `content` key etc.). Consider wrapping the whole function body in a `try…catch` block to handle any errors.